src/dbnode/persist/fs/seek_manager_test.go

```diff
@@ -73,7 +73,9 @@ func TestSeekerManagerCacheShardIndices(t *testing.T) {
 	// Assert captured byTime objects match expectations
 	require.Equal(t, len(shards), len(byTimes))
 	for _, shard := range shards {
+		mu.Lock()
```

Comment:
need to guard against concurrent access to `byTimes` with func on line 66 which runs in a goroutine